desktop/scripts/get-tor.py

```diff
@@ -286,46 +286,46 @@ def update_tor_bridges():
     torrc_template_dir = os.path.join(
         root_path, os.pardir, "cli/onionshare_cli/resources"
     )
-    endpoint = "https://bridges.torproject.org/moat/circumvention/builtin"
```

Comment:
I've raised https://gitlab.torproject.org/tpo/anti-censorship/rdsys/-/issues/122 to ask about this.